src/content/docs/contributing/nf_core_basic_training.md

```diff
@@ -30,64 +45,72 @@ The course is going to build an (totally unscientific and useless) RNA seq pipel
 
 The following sections will be handled in the course:
 
-**1. Setting up the gitpod environment for the course**
+1. **Setting up the gitpod environment for the course**
+
+  The course is using gitpod in order to avoid the time expense for downloading and installing tools and data.
+
+2. **Exploring the nf-core tools command**
 
-The course is using gitpod in order to avoid the time expense for downloading and installing tools and data.
+  A very basic walk-through of what can be done with nf-core tools
 
-**2. Exploring the nf-core tools command**
+3. **Creating a new nf-core pipeline from the nf-core template**
 
-A very basic walk-through of what can be done with nf-core tools
+4. **Exploring the nf-core template**
 
-**3. Creating a new nf-core pipeline from the nf-core template**
+  a) The git repository
 
-**4. Exploring the nf-core template**
+  b) running the pipeline
 
-a) The git repository
+  c) linting the pipeline
 
-b) running the pipeline
+  d) walk-through of the template files
 
-c) linting the pipeline
+5. **Building a nf-core pipeline using the template**
 
-d) walk-through of the template files
+  a) Adding a nf-core module to your pipeline
 
-**5. Building a nf-core pipeline using the template**
+  b) Adding a local custom module to your pipeline
 
-a) Adding a nf-core module to your pipeline
+  c) Working with Nextflow schema
 
-b) Adding a local custom module to your pipeline
+  d) Linting your modules
```

Comment:
somehow the indentation for the sub-points is messed up, which makes this section very hard to read.